CtfPlayback/Metadata/TypeInterfaces/ICtfFloatingPointDescriptor.cs

```diff
@@ -4,9 +4,9 @@
 namespace CtfPlayback.Metadata.TypeInterfaces
 {
     /// <summary>
-    /// This interface is not yet implemented
+    /// Describes the properties of a floating point type.
     /// </summary>
-    public interface ICtfFloatDescriptor
+    public interface ICtfFloatingPointDescriptor
```

Comment:
>ICtfFloatingPointDescriptor [](start = 21, length = 27)

just a heads up... I *believe* this is a generated file from the antlr grammar. so you'll either have to make the corresponding change in the grammar, or you'll need to rename this file any time the grammar is updated going forward.

Reply:
I also could be wrong, but I don't think this is auto-generated. It doesn't have that signature/remarks and none of the auto-generated code refers to this. Also not called out in grammer\ctf.cmd